django_mongodb_backend/encryption.py

```diff
@@ -0,0 +1,66 @@
+# Queryable Encryption helpers
+#
+# TODO: Decide if these helpers should even exist, and if so, find a permanent
+# place for them.
+
+from bson.binary import STANDARD
+from bson.codec_options import CodecOptions
+from pymongo.encryption import AutoEncryptionOpts, ClientEncryption
+
+
+def get_encrypted_client(auto_encryption_opts, encrypted_connection):
+    """
+    Returns a `ClientEncryption` instance for MongoDB Client-Side Field Level
+    Encryption (CSFLE) that can be used to create an encrypted collection.
+    """
+
+    key_vault_namespace = auto_encryption_opts._key_vault_namespace
+    kms_providers = auto_encryption_opts._kms_providers
+    codec_options = CodecOptions(uuid_representation=STANDARD)
+    return ClientEncryption(kms_providers, key_vault_namespace, encrypted_connection, codec_options)
+
+
+def get_auto_encryption_opts(crypt_shared_lib_path=None, kms_providers=None):
```

Comment:
`crypt_shared` library is in the wheel too which is much easier to use than downloading separate and telling `MongoClient` where it is.

Reply:
More to this story:
- `libmongocrypt` is in the `pymongocrypt` wheel, not `crypt_shared` which must always be downloaded and configured manually.
- `libmongocrypt` works because `mongocryptd` is running on enterprise. 

(via @ShaneHarvey, thanks!)